docs/AvailabilitySubscriber.md

```diff
@@ -48,6 +48,12 @@ When implementing this component as a block, various inner blocks may be availab
   }
 ```
 
+| Prop name            | Type      | Description                                                                      | Default value      |
+| -------------------- | --------- | -------------------------------------------------------------------------------- | ------------------ |
+| `namePlaceholder`      | `string` | text value of the name input placeholder.  |  undefined         |
```

Comment:
```suggestion
| `namePlaceholder`      | `string` | Text value of the name input placeholder.  |  `undefined`         |
```

Reply:
Wait, the default value is not `undefined`, right? The default value here should be the default placeholder displayed by the component today